component/custom/custom_loader.go

```diff
@@ -0,0 +1,67 @@
+package custom
+
+import (
+	"os"
+	"path/filepath"
+	"plugin"
+	"strings"
+
+	"github.com/Azure/azure-storage-fuse/v2/common/log"
+	"github.com/Azure/azure-storage-fuse/v2/external"
+	"github.com/Azure/azure-storage-fuse/v2/internal"
+)
+
+func init() {
+	// Environment variable for directory containing the plugin .so files.
+	pluginDirPath := os.Getenv("BLOBFUSE_PLUGIN_DIRECTORY")
+	if pluginDirPath == "" {
+		return
+	}
+
+	// Environment variable which expects file names as a colon-separated list of `.so` files located in
+	// the specified plugin directory. For example: BLOBFUSE_PLUGIN_NAMES=plugin1.so:plugin2.so:plugin3.so
+	pluginFilesPath := os.Getenv("BLOBFUSE_PLUGIN_NAMES")
```

Comment:
Do we need two env variables for this. Instead of path we could have asked user to provide fully qualified path in this variable itself seperated by ":" or something

Reply:
Ideally in these plugin models there should be just one specific folder for .so files. As a good practice used two variables for one folder and another to specify the file names. Open for suggestions on this.